### PR TITLE
Issue no 88 is resolved when instanceId is already present in endpoin…

### PR DESCRIPTION
**Pull request by amitrnec**
When instance-Id is already present in endpoint,It should not be updated.

### CODE REVIEW

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java [hunk 1]
```diff
@@ -295,8 +295,10 @@ public void modifyAttribInstanceTemporalEntity(String entityId, String payload,
 
 		// get original createdAt
 		String createdAt = now;
+		String instanceIdAdd = null;
+		final JsonArray jsonArray = parser.parse(historyDAO.getListAsJsonArray(entityList)).getAsJsonArray();
```
Comment (naveennec) [inline History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java:299]: now if parser give exception in any case, then request will be crashed which is not expected. better to write back in try block.

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java [hunk 2]
```diff
@@ -335,6 +337,11 @@ public void modifyAttribInstanceTemporalEntity(String entityId, String payload,
 							throw new ResponseException(ErrorType.InvalidRequest,
 									"instanceId in payload and in URL must be the same");
 						}
+					} else {
+						instanceIdAdd = jsonArray.get(0).getAsJsonObject().get(resolvedAttrId).getAsJsonArray().get(0).getAsJsonObject()
+								.get(NGSIConstants.NGSI_LD_INSTANCE_ID).getAsJsonArray().get(0).getAsJsonObject()
+								.get(NGSIConstants.JSON_LD_ID).getAsString();
+								jsonElement = setTemporalPropertyinstanceId(jsonElement, NGSIConstants.NGSI_LD_INSTANCE_ID, instanceIdAdd);
```
Comment (naveennec) [inline History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java:344]: code formatting is not proper, try to format it in eclipse only then commit the code.

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java (hunk 1 above)
Comment (amitrnec) [reply History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java]: done

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java (hunk 2 above)
Comment (amitrnec) [reply History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/service/HistoryService.java]: done

**Comment by naveennec** [comment]
Done basic testing , looks fine to me

**Pull request merged**